### PR TITLE
8 packages from ocaml/opam at 2.0.5

### PR DESCRIPTION
**Pull request by rjbou**
This pull-request concerns:
-`opam-client.2.0.5`
-`opam-core.2.0.5`
-`opam-devel.2.0.5`
-`opam-format.2.0.5`
-`opam-installer.2.0.5`
-`opam-repository.2.0.5`
-`opam-solver.2.0.5`
-`opam-state.2.0.5`

---
* Source repo: git+https://github.com/ocaml/opam.git
* Bug tracker: https://github.com/ocaml/opam/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/opam-repository/opam-repository.2.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "Anil Madhavapeddy <anil@recoil.org>"
+  "Guillem Rieu <guillem.rieu@ocamlpro.com>"
+  "Ralf Treinen <ralf.treinen@pps.jussieu.fr>"
+  "Frederic Tuong <tuong@users.gforge.inria.fr>"
+]
+homepage: "https://opam.ocaml.org/"
+bug-reports: "https://github.com/ocaml/opam/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "opam-format" {= "2.0.5"}
```
Comment (kit-ty-kate) [inline packages/opam-repository/opam-repository.2.0.5/opam]: for next time, I think this pattern can be improved to `{= version}`

**Comment by kit-ty-kate** [comment]
Somehow it seems `opam-devel`'s tests fail on Alpine: https://ci.ocaml.org/log/saved/docker-build-0c2886844a839be56133521d8ec5a863/cb9814689228f6a1477695b2b09e4f4e95beecf2

**Comment by kit-ty-kate** [comment]
Sounds ok otherwise. I can merge this one once the Alpine failure has been cleared up (whether it's fine as is or needs fixing)

File: packages/opam-repository/opam-repository.2.0.5/opam (hunk 1 above)
Comment (rjbou) [reply packages/opam-repository/opam-repository.2.0.5/opam]: yep! It would decrease the number of changes also :)

**Comment by rjbou** [comment]
How can I test locally the alpine? Is the generated or generator of dockerfile available somewhere ? (I always forget where...)

**Comment by avsm** [comment]
@rjbou it's at https://ci.ocaml.org/log/saved/opam-build-622f9f3b88756a2f6c726ffaef063f64/458766cc227902d82eeda3422b1912b8888db7d0 (from surfing the https://ci.ocaml.org/ocaml/opam-repository/pr/14481 view and looking for the "Dockerfile opam-client.2.0.5 <..>" job that generates it)

**Comment by rjbou** [comment]
alpine is the only that launches tests. As internal test suite checks system compiler and there isn't installed on these images, it's the only one that fails.

**Comment by avsm** [comment]
Thanks; I've got it in on my list to make a system compiler available in the next iteration of the images. Not a blocker to merge this for now :)

**Comment by avsm** [comment]
Just these warnings: shouldnt we make the opam package `opam lint` clean? :-)

warning 47: Synopsis (or description first line) should start with a capital and not end with a dot

**Comment by rjbou** [comment]
Ideally yes, but as opam synopses begin with `opam` that doesn't capitalize...

**Comment by RalfJung** [comment]
> Ideally yes, but as opam synopses begin with opam that doesn't capitalize...

Sounds like an overeager lint to me then?^^

**Comment by rjbou** [comment]
> Sounds like an overeager lint to me then?^^

That's what help us to have consistent & nice view of [packages lists](https://opam.ocaml.org/packages) (as you can notice, it's not homogeneous about the ending dot).
Plus, cases when sentence begins with a non capitalized program name are minimal compared to forgetting a capital at the beginning ; the last one being noticed & fixed because of the lint warning.

**Comment by avsm** [comment]
Thanks, not a blocker to merge. I do think that it's worth aiming for making `opam lint` unconditionally work without warnings somehow (especially for corner cases like this one). It's frustrating to have repository wide warnings that can't be suppressed and adding noise to the real issues.

**Pull request merged**